beluga/include/beluga/sensor/ndt_sensor_model.hpp

```diff
@@ -161,9 +160,9 @@ using NDTModelParam3d = NDTModelParam<3>;
 template <typename SparseGridT>
 class NDTSensorModel {
  public:
-  static_assert(std::is_same_v<SparseValueGrid<typename SparseGridT::map_type>, SparseGridT>);
   /// NDT Cell type.
   using ndt_cell_type = typename SparseGridT::mapped_type;
+  static_assert(std::is_same_v<SparseValueGrid<typename SparseGridT::map_type, ndt_cell_type::num_dim>, SparseGridT>);
```

Comment:
@serraramiro1 meta: not to be addressed now, but if we know this must be an specialization of `SparseValueGrid`, why take it as a template parameter instead of taking its constituent parts?